samples/login-sample/src/main/java/com/uber/sdk/android/samples/LoginSampleActivity.java

```diff
@@ -179,9 +213,23 @@ public void onLoginSuccess(@NonNull AccessToken accessToken) {
 
         @Override
         public void onAuthorizationCodeReceived(@NonNull String authorizationCode) {
-            Toast.makeText(LoginSampleActivity.this, getString(R.string.authorization_code_message, authorizationCode),
-                    Toast.LENGTH_LONG)
-                    .show();
+            new AuthorizationCodeGrantFlow(loginManager.getSessionConfiguration(),
+                    authorizationCode,
+                    CODE_VERIFIER).execute(new TokenRequestFlow.TokenRequestFlowCallback() {
+                @Override
+                public void onSuccess(AccessToken accessToken) {
+                    new AccessTokenManager(LoginSampleActivity.this).setAccessToken(accessToken);
```

Comment:
You'd probably want your abstract I mentioned above to handle this, as opposed to the user doing it in callback

Reply:
All the changes have been made assuming the developer is not integrating the SDK rather using the assembling applink directly in their app and launching it.

Reply:
Got it, then I think this is a bit confusing and doesn't communicate usage well as the example here still uses the SDK to acccess the TokenManager, Assemberl, and other classes, adds some custom logic, and intermixes a different usage.

If you're intention is to have an example that could entirely get away from using the SDK, then it's probably worth doing a second sample app here without the dep on the main SDK and possibly extracting the needed classes into a separate module (separate deploying maven artifact) that they can depend on without the rest.

Reply:
fair point. 
i will update the current diff with sdk usage and put up a second PR for another sample app demo-ing usage without android sdk